### PR TITLE
[Merged by Bors] - style: whitespace and long line

### PR DESCRIPTION
**Pull request by adomani**
One of these was mentioned in the 100 vs 101 character lint, but I forgot about it.

The shortening of the line in `Mathlib/Order/Rel/GaloisConnection.lean` was flagged by the long line linter.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — COMMENTED** [review]
I'm in favour - thanks for doing this!
maintainer merge

File: Mathlib/Computability/AkraBazzi/AkraBazzi.lean [hunk 1]
```diff
@@ -585,8 +585,8 @@ lemma asympBound_def' {n : ℕ} :
   simp [asympBound_def, sumTransform, mul_add, mul_one, Finset.sum_Ico_eq_sum_range]
 
 lemma asympBound_pos (n : ℕ) (hn : 0 < n) : 0 < asympBound g a b n := by
-  calc 0 < (n : ℝ) ^ p a b * (1 + 0)   := by aesop (add safe Real.rpow_pos_of_pos)
-       _ ≤ asympBound g a b n    := by
+  calc 0 < (n : ℝ) ^ p a b * (1 + 0) := by aesop (add safe Real.rpow_pos_of_pos)
```
Comment (mattrobball) [inline Mathlib/Computability/AkraBazzi/AkraBazzi.lean:588]: What is the alignment convention for proofs in `calc`? We seem to have both here.

File: Mathlib/Order/Rel/GaloisConnection.lean [hunk 2]
```diff
@@ -58,7 +58,8 @@ theorem gc_leftDual_rightDual : GaloisConnection (toDual ∘ R.leftDual) (R.righ
 /-- `leftFixedPoints` is the set of elements `J : Set α` satisfying `rightDual (leftDual J) = J`. -/
 def leftFixedPoints := {J : Set α | R.rightDual (R.leftDual J) = J}
 
-/-- `rightFixedPoints` is the set of elements `I : Set β` satisfying `leftDual (rightDual I) = I`. -/
+/-- `rightFixedPoints` is the set of elements `I : Set β` satisfying `leftDual (rightDual I) = I`.
```
Comment (mattrobball) [inline Mathlib/Order/Rel/GaloisConnection.lean:61]: Personally, I'd like `/--` on its own line but I don't know the community's sentiment.

Comment (grunweg) [reply Mathlib/Order/Rel/GaloisConnection.lean:61]: My understanding of it is "on a separate line is fine for single sentences/up to two lines; separate paragraphs implies delimiters on separate lines". Basically, close Lean core style (but allowing
```
/-- Two lines of 
doc comment -/
```
This one seems to be in accordance with it.

File: Mathlib/Computability/AkraBazzi/AkraBazzi.lean (hunk 1 above)
Comment (grunweg) [reply Mathlib/Computability/AkraBazzi/AkraBazzi.lean:588]: adomani/linter_stain_active

File: Mathlib/Order/Rel/GaloisConnection.lean (hunk 2 above)
Comment (adomani) [reply Mathlib/Order/Rel/GaloisConnection.lean:61]: I tried to maintain the symmetry with the one above, that is why I went for a minimalistic change.

**Comment by mattrobball** [comment]
bors merge

**Pull request closed without merging**